lib/mix/lib/mix/tasks/compile.elixir.ex

```diff
@@ -47,6 +50,14 @@ defmodule Mix.Tasks.Compile.Elixir do
   def run(args) do
     {opts, _, _} = OptionParser.parse(args, switches: @switches)
 
+    # The :long_compilation_threshold is in seconds, but we want to pass it to
+    # the compiler in milliseconds.
+    opts =
+      case Keyword.fetch(opts, :long_compilation_threshold) do
```

Comment:
`Keyword.get(options, :long_compilation_threshold) || 5_000` –> `options[:long_compilation_threshold] || 5_000`
 
Either looks okay, I think if we keep current implementation we should use `Keyword.get/3` in https://github.com/elixir-lang/elixir/pull/4628/files#diff-59fd3b02a2e095f27bfc670699154054R115.